tests/requirements-relaxed.txt

```diff
@@ -14,15 +14,17 @@ aiosignal==1.3.1
     # via aiohttp
 alabaster==0.7.16
     # via sphinx
+annotated-types==0.6.0
+    # via pydantic
 ansible-pygments==0.1.1
     # via
     #   antsibull-docs
     #   sphinx-ansible-theme
-antsibull-core==2.2.0
+antsibull-core==3.0.0
     # via antsibull-docs
-antsibull-docs==2.7.0
+antsibull-docs==2.9.0
     # via -r tests/requirements-relaxed.in
-antsibull-docs-parser==1.0.0
+antsibull-docs-parser==1.0.1
```

Comment:
This change just contains a fix for markdown parsing which we don't use, so nothing to worry about there.